src/modules/editor/ScriptEditorImplementation.cpp

```diff
@@ -457,8 +457,9 @@ bool ScriptEditorWidget::contextSensitiveHelp() const
 	QTextCursor cur = cursorForPosition(QPoint(r.x(), r.y()));
 	cur.select(QTextCursor::WordUnderCursor);
 	QString szText = cur.selectedText();
-	QString szTmp = szText;
-
+	KviQString::escapeKvs(&szText);
+	QString szParse = QString("timer -s (help,0){ help.open %1; }").arg(szText);
```

Comment:
Why does it need to go through timer?

Reply:
That's how it worked in the past:
https://github.com/kvirc/KVIrc/commit/5f0292c32b336be0bb279a7511a9ee37ba9df3fa#diff-ee6389e2178d7b15aa4a4ccc1699f64becf9a3e25feedcfde4e4ea6d0f2905eaL539

I guess the idea was to make it async